docs/troubleshooting.md

```diff
@@ -0,0 +1,48 @@
+# Troubleshooting
+
+## Typical errors
+
+An error occurred during namespace creation: Unauthorized
```

Comment:
In my opinion it`s will be more visible if you put there subsection symbol '###'.

In other errors too.